extensions/ql-vscode/src/extension.ts

```diff
@@ -1010,6 +1013,33 @@ async function activateWithInstalledDistribution(
   };
 }
 
+/**
+ * Handle changes to the external config file. This is used to restart the query server
+ * when the user changes options.
+ * See https://docs.github.com/en/code-security/codeql-cli/using-the-codeql-cli/specifying-command-options-in-a-codeql-configuration-file#using-a-codeql-configuration-file
+ */
+function watchExternalConfigFile(app: ExtensionApp, ctx: ExtensionContext) {
+  if (process.env.HOME) {
+    const configPath = join(process.env.HOME, ".config/codeql", "config");
```

Comment:
I think it would be better to be consistent in how we join here:

```suggestion
    const configPath = join(process.env.HOME, ".config", "codeql", "config");
```